de.metas.shipper.gateway.dpd/src/test/java/de/metas/shipper/gateway/dpd/logger/DpdClientLogEventTest.java

```diff
@@ -0,0 +1,70 @@
+
+		assertEquals(expectedString, logEvent.cleanupPdfData(originalString));
+	}
+
+	@Test
+	void testRegexPdfNotRemoved_StoreOrderResponse()
+	{
+		final String originalString = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><ns3:storeOrdersResponse xmlns:ns2=\"http://dpd.com/common/service/types/Authentication/2.0\" xmlns:ns4=\"http://dpd.com/common/service/types/LoginService/2.0\" xmlns:ns3=\"http://dpd.com/common/service/types/ShipmentService/3.2\"><orderResult><parcellabelsPDF>JVBERi0xLjQKJeLjz9MKMyAwIG9iago8PC9EZWNvZGVQYXJtczw8L0NvbG9ycyAzL1ByZWRpY3RvciAxNS9CaXRzUGVyQ29tcG9uZW50IDgvQ29sdW1ucyAxNDg+Pi9UeXBlL1hPYmplY3QvQ29sb3JTcGFjZS9EZXZpY2VSR0IvU3VidHlwZS9JbWFnZS9CaXRzUGVyQ29tcG9uZW50IDgvV2lkdGggMTQ4L0xlbmd0aCAxNjA1L0hlaWdodCAxNDgvRmlsdGVyL0ZsYXRlRGVjb2RlPj5zdHJlYW0KeNrtncGO20AMQ/P/P90CPRVYxBb56GRHoE9GN3XG87wyRaro60+PY49Xt6DwehRejwi817/j5/n1539+8t113n3y3aFe83rN764//5Pr9UyuYwL7/5qFtxbe9dLnUOc/Jds3wa9CvX6A5p/3Hr6buy68VfDU7Zsvd7KJpJyS9aR+6pXl+fcW3nZ4Pf/N54W3C97kF3YuCiZ/a15MJjI9JZqeuL4nYW5ePYW3Cl62TSYvdiJk5g/Q/O7m96heIdAqFN7BNDdmODdhYjJiNDE1YjA4ZjFlZDhmM2E1ZTk3YzQ+PGU2NjNmZjA1NzEzMzdlMDk4MDJkMDdhODFlZjFiMzYzPl0vSW5mbyA1NyAwIFIvU2l6ZSA1OD4+CnN0YXJ0eHJlZgoxMzE5ODQKJSVFT0YK</parcellabelsPDF><shipmentResponses><identificationNumber>100003</identificationNumber><mpsId>EXP0998505274844120191122</mpsId><parcelInformation><parcelLabelNumber>09985052748441</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748442</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748443</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748444</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748445</parcelLabelNumber></parcelInformation></shipmentResponses></orderResult></ns3:storeOrdersResponse>";
+		final String expectedString = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><ns3:storeOrdersResponse xmlns:ns2=\"http://dpd.com/common/service/types/Authentication/2.0\" xmlns:ns4=\"http://dpd.com/common/service/types/LoginService/2.0\" xmlns:ns3=\"http://dpd.com/common/service/types/ShipmentService/3.2\"><orderResult><parcellabelsPDF>PDF TEXT REMOVED!</parcellabelsPDF><shipmentResponses><identificationNumber>100003</identificationNumber><mpsId>EXP0998505274844120191122</mpsId><parcelInformation><parcelLabelNumber>09985052748441</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748442</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748443</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748444</parcelLabelNumber></parcelInformation><parcelInformation><parcelLabelNumber>09985052748445</parcelLabelNumber></parcelInformation></shipmentResponses></orderResult></ns3:storeOrdersResponse>";
+
+		assertEquals(expectedString, logEvent.cleanupPdfData(originalString));
```

Comment:
expectedString is not needed. use originalString. it would make the test even more readable.

Reply:
Good point, thanks :)